docs/sphinx/about/whats-new.txt

```diff
@@ -1,6 +1,14 @@
 Version history
 ===============
 
+5.3.1 (2016 December 16)
+-----------------------
+
+Java bug fixes:
+
+* TIFF
+   - fixed invalid IFD values when writing TIFF files with ``loci.formats.TiffWriter``
```

Comment:
Might be worth precising (OME-)TIFF as both formats are affected. Do we need to precise the affected utilities like `bfconvert`?